selene-vscode/src/extension.ts

```diff
@@ -26,10 +26,24 @@ enum Severity {
     Warning = "Warning",
 }
 
+function byteToCharOffset(document: vscode.TextDocument, byteOffset: number) {
+    const text = document.getText();
+    let currentOffset = 0;
+    // Iterate through each character in the string
+    for (let i = 0; i < byteOffset; i++) {
```

Comment:
Use `byte` instead of `i`.

Reply:
This is an iteration over the file characters, would `char` be more suitable here?

Reply:
Sure, both are more descriptive than `i` (though you should prefer `character`).

Reply:
I'd use something like `charOffset` though, I'm not sure what this is but I know it's a number, and a variable named just `char` should refer to a string.